rpcd/playbooks/roles/rpc_maas/tasks/rabbitmq_user.yml

```diff
@@ -25,3 +25,31 @@
     state: "present"
   tags:
     - rabbitmq-user
+
+  # uri module not used because httplib2 not available in the rabbit container
```

Comment:
Could you add something like: "TODO: move back to uri when ansible 2.1 is officially supported for RPC"